omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -58,6 +59,31 @@ def solr_root
     host
   end
 
+  def elastic_search_major_version
```

Comment:
That's a good point...  Should we raise exceptions and fail hard for some of these?